.freebsdci.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+# FreeBSD CI Build Scripts
+# The flow of a FreeBSD CI (https://freebsdci.julialang.org) build:
+#
+# 1. `compile`
+# 2. `build-state`
+# 3. `runtests`
+# 4. `test-embedding`
+#
+# Detail of flow is controlled by the variable `factory`
+# here.
+# https://github.com/iblis17/julia-fbsd-buildbot/blob/master/master/master.cfg
+#
+# Usage: .freebsdci.sh <stage>
+
+set -xe
+
+build-state(){
+    gmake build-stats
+}
+
+compile(){
+    export MALLOC_CONF='junk:false'
+    export VERBOSE=1
+    export FORCE_ASSERTIONS=1
+    export LLVM_ASSERTIONS=1
+
+    gmake all -j $MAKE_JOBS_NUMBER
+}
+
+runtests(){
+    export MALLOC_CONF='junk:false'
+    export VERBOSE=1
+    export FORCE_ASSERTIONS=1
+    export LLVM_ASSERTIONS=1
+    export JULIA_TEST_MAXRSS_MB=600
+    export JULIA_CPU_CORES=$MAKE_JOBS_NUMBER
+
+    gmake testall \
+        test-download \
+        test-Pkg \ # <- TODO: Should only test Pkg/pkg but no rule for test-Pkg/pkg
```

Comment:
The comment in this line neutralized the effect of `\` , so FreeBSD CI build failed.

line 73:
https://freebsdci.julialang.org/#/builders/1/builds/7031/steps/7/logs/stdio

Reply:
Oops, thanks.

Reply:
Is there anyway we could run this as `julia test/runtests.jl test-Pkg/pkg`?

Reply:
I think invoking `./usr/bin/julia` should work